ui/src/app/components/object-detail/object-detail.component.html

```diff
@@ -0,0 +1,451 @@
+                      [disabled]="element.get('spName').value === ''"
+                      mat-menu-item
+                      (click)="dropFk(element)"
+                    >
+                      <span>Drop Fk</span>
+                    </button>
+                  </mat-menu>
+                </div>
+              </div>
+            </td>
+          </ng-container>
+
+          <tr mat-header-row *matHeaderRowDef="['srcDatabase', 'spDatabase']"></tr>
+          <tr mat-header-row *matHeaderRowDef="displayedFkColumns"></tr>
+          <tr mat-row *matRowDef="let row; columns: displayedFkColumns"></tr>
+        </table>
+      </div>
+    </mat-tab>
+
+    <mat-tab label="SQL">
+      <div class="ddl_container">
+        <pre>
+          <code>{{ddlStmts[currentObject!.name]}}</code>
+        </pre>
+      </div>
+    </mat-tab>
+  </mat-tab-group>
+
+  <div *ngIf="currentObject!.type === 'indexName'" class="column-tab-container">
+    <table mat-table [dataSource]="dataSource">
+      <ng-container matColumnDef="srcDatabase">
+        <th mat-header-cell class="db_name" *matHeaderCellDef colspan="2">{{ srcDbName }}</th>
+      </ng-container>
+      <ng-container matColumnDef="spDatabase">
+        <th mat-header-cell *matHeaderCellDef class="db_name" colspan="2">
+          <div class="spanner_edit-button">
+            <span>Spanner</span>
+          </div>
+        </th>
+      </ng-container>
+      <ng-container matColumnDef="srcOrder">
+        <th class="table_header" mat-header-cell *matHeaderCellDef>Sort Order</th>
+        <td mat-cell *matCellDef="let element">
+          {{ element.get('srcOrder').value }}
+        </td>
+      </ng-container>
+
+      <ng-container matColumnDef="srcColName">
+        <th class="table_header" mat-header-cell *matHeaderCellDef>Column</th>
+        <td mat-cell *matCellDef="let element">
+          {{ element.get('srcColName').value }}
+        </td>
+      </ng-container>
+
+      <ng-container matColumnDef="spOrder">
+        <th class="table_header" mat-header-cell *matHeaderCellDef>Sort Order</th>
+        <td mat-cell *matCellDef="let element">
+          {{ element.get('spOrder').value }}
+        </td>
+      </ng-container>
+
+      <ng-container matColumnDef="spColName">
+        <th mat-header-cell class="table_header" *matHeaderCellDef>Column</th>
+        <td mat-cell *matCellDef="let element">
+          {{ element.get('spColName').value }}
+        </td>
+        <tr mat-header-row *matHeaderRowDef="['srcDatabase', 'spDatabase']"></tr>
+        <tr mat-header-row *matHeaderRowDef="indexDisplayedColumns"></tr>
+        <tr mat-row *matRowDef="let row; columns: indexDisplayedColumns"></tr>
+      </ng-container>
+    </table>
+  </div>
+
+  <div class="save-edits">
+    <mat-divider></mat-divider>
+    <div class="save-button" *ngIf="isEditMode && currentTabIndex === 0">
+      <button mat-raised-button color="primary" (click)="toggleEdit()">SAVE & CONVERT</button>
```

Comment:
Review changes page is not getting rendered after clicking on save & convert. Is this expected behaviour?

Reply:
We've added this in the backlog list.